packages/table/components/Column.tsx

```diff
@@ -5,6 +5,8 @@ export interface IWidthArgs {
   remainingWidth: number;
   currentIndex: number;
 }
+
+export type Width = string | number | undefined;
```

Comment:
I think this abstraction has a problem. As a cell renderer I would need to know in which table I am used and if this table uses a string width like `10%` or a numeric pixel value. I think we should only work with numbers here, anything else provides an extra layer of complexity with no real benefit on the user-side